modules/appeals_api/spec/docs/decision_reviews/supplemental_claims_spec.rb

```diff
@@ -359,7 +359,7 @@ def swagger_doc
       tags 'Supplemental Claims'
       operationId 'putSupplementalClaimEvidenceSubmission'
 
-      description File.read(AppealsApi::Engine.root.join('app', 'swagger', 'supplemental_claims', 'v0', 'put_description.md'))
+      description File.read(AppealsApi::Engine.root.join('app', 'swagger', 'decision_reviews', 'v2', 'put_description.md'))
```

Comment:
These two `put_description.md` files used to be identical, so I hadn't noticed this mistake - this is now the correct path